web/pdf_viewer.css

```diff
@@ -15,6 +15,11 @@
 @import url(text_layer_builder.css);
 @import url(annotation_layer_builder.css);
 
+.pdfViewer, .spread {
```

Comment:
Isn't this block only necessary for `.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped` instead of `.pdfViewer`? In vertical scrolling mode there is only one page per row, so this selector seems overly broad.

The vertical scrolling + spreads case is already covered by `.spread`.